src/environment.go

```diff
@@ -185,7 +186,8 @@ func (env *environment) getcwd() string {
 	}
 	correctPath := func(pwd string) string {
 		// on Windows, and being case sensitive and not consistent and all, this gives silly issues
-		return strings.Replace(pwd, "c:", "C:", 1)
+		driveLetter := regexp.MustCompile(`^[a-z]:`)
```

Comment:
you can use getCompiledRegex(`^[a-z]:`) here. We have functionality that cases the regex once compiled.